lib/wallet/test/unit/Cardano/Wallet/DB/Store/Transactions/StoreSpec.hs

```diff
@@ -149,3 +156,11 @@ genDeltas (TxSet pile) =
                     else elements (Map.keys pile)
             )
         ]
+
+prop_getTxById :: StoreProperty
+prop_getTxById = withStoreProp $ \runQ -> do
+    txs :: TxSet <- liftIO $ generate $ (`apply` mempty) <$> genDeltas mempty 
```

Comment:
```suggestion
genTxSet :: Gen TxSet
genTxSet = (`apply` mempty) <$> genDeltas mempty 

prop_getTxById :: StoreProperty 
prop_getTxById = forAll genTxSet $ \txs ->
    withStoreProp $ \runQ -> do
```
(module type of `withStoreProp`).

I believe that `liftIO . generate` generates only a single test case, rather than multiple ones — `pick` would be better.

In any case, it is good style to formulate QuickCheck properties as logical statements: "For all `TxSet`, storing the set in the database and calling `getTxById` on every id will return the expected transaction." Randomized test case generation is QuickCheck's way of interpreting "for all", but from the viewpoint of language, we really want to express that "For all `TxSet`, … is true" (though QuickCheck only checks a minuscule fraction of "all").